addons/account/views/account_tax_views.xml

```diff
@@ -249,9 +249,9 @@
                     <field name="company_id" groups="base.group_multi_company"/>
                     <field name="country_code" column_invisible="True"/>
 
-                    <field name="tax_payable_account_id" domain="[('company_id', '=', company_id)]"/>
-                    <field name="tax_receivable_account_id" domain="[('company_id', '=', company_id)]"/>
-                    <field name="advance_tax_payment_account_id" domain="[('company_id', '=', company_id)]"/>
+                    <field name="tax_payable_account_id" domain="[('company_ids', '=', company_id)]"/>
```

Comment:
This should use `check_company` instead

Reply:
Also there already:
https://github.com/odoo/odoo/blob/f02ffa829e14886df23a8f9b7b5688808fcb7e8c/addons/account/models/account_tax.py#L35-L39